drivers/periph_common/init_leds.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_periph_init
+ * @{
+ *
+ * @file
+ * @brief       Init on board LEDs
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ * @}
+ */
+
+#include "board.h"
+#include "periph/gpio.h"
+#include "kernel_defines.h"
+
+#define LED_INIT(x) do {                                            \
+                        if (IS_ACTIVE(MODULE_PERIPH_INIT_LED##x)) {   \
+                            gpio_init(LED##x##_PIN, GPIO_OUT);      \
+                            LED##x##_OFF;                           \
+                        }                                           \
+                    } while (0)
+
+void leds_int(void)
```

Comment:
What does `leds_int` mean ???

Reply:
> What does `leds_int` mean ???

I think this may have been the solution to avoid conflicts no @benpicco?

Reply:
@benpicco if this was because of a name conflict maybe we can just call the function `led_init`?

Reply:
or leds_setup()

Reply:
oops, typo - it should spell `leds_init()`